src/AppRoutes.jsx

```diff
@@ -0,0 +1,25 @@
+import { BrowserRouter, Routes, Route } from 'react-router-dom';
+import HomePage from './components/pages/HomePage.jsx';
+import NotFound from './components/pages/NotFound.jsx';
+import Login from './components/auth/Login.jsx';
+import Register from './components/auth/Register.jsx';
+import SearchBar from './components/search/SearchBar.jsx';
+import PaymentForm from './components/payment/PaymentForm.jsx';
+
+
+const AppRoutes = () => {
+  return (
+    <BrowserRouter>
+      <Routes>
+        <Route path="*" element={<NotFound />} />
```

Comment:
Good job add this "catch-all" route. But there could be some issues with this route being placed before all other routes, since its intention is to catch all paths. Move this to the bottom.

Reply:
https://ui.dev/react-router-handling-404-pages
With router version 6 we van put that rout first